.gitignore

```diff
@@ -13,3 +13,11 @@ backend/openapi.json
 
 # ignore jupyter notebook checkpoints
 .ipynb_checkpoints
+
+# edit docs using obsidian.md, these files should not appear in the repo
+docs/docs/.obsidian/workspace.json
```

Comment:
I think instead of all these you can just have one line: `.obsidian/`

Reply:
done